### PR TITLE
adds a changelog

### PR DESCRIPTION
**Pull request by faiq**
you know, for releases and stuff

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## 2016.02.10
+
+### Features
+
+- Allow users to mount local volumes to thier wercker build containers
+- Check to see if config from wercker.yml is empty
+
```
Comment (termie) [inline changelog.md]: - Adds changelog

**Comment by termie** [comment]
looks good, i have one meta request, but otherwise lets do it

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## 2016.02.10
+
+### Features
+
+- Allow users to mount local volumes to thier wercker build containers
```
Comment (termie) [inline changelog.md]: spelling/typos their

Comment (termie) [inline changelog.md]: oh and maybe say "with the --enable-volumes flag"

Comment (termie) [inline changelog.md]: and the volumes field in the box description

Comment (faiq) [inline changelog.md]: i added volumes field in box description. the rest should be for doc changes

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## 2016.02.10
+
+### Features
+
+- Allow users to mount local volumes to their wercker build containers, specified by a list of `volumes` underneath box in the werker.yml file
```
Comment (termie) [inline changelog.md]: "and enabled by the --enable-volumes flag "

**Pull request merged**